pkg/dnsserver/dnscontroller/handler.go

```diff
@@ -0,0 +1,43 @@
+package dnscontroller
+
+import (
+	"net"
+	"strings"
+
+	"k8s.io/klog"
+
+	"github.com/miekg/dns"
+)
+
+func (lh Lighthouse) ServeDNS(w dns.ResponseWriter, r *dns.Msg) {
+	klog.Errorf("Serving  DNSRequest")
+	msg := dns.Msg{}
+	msg.SetReply(r)
```

Comment:
Please add a newline here and to separate logical sections below to improve readability.